quint/testFixture/SuperSpec.qnt

```diff
@@ -179,6 +179,7 @@ module withConsts {
   val test_list_nth = [2, 3, 4][2]
   val test_record = { name: "igor", year: 1981 }
   val test_record2 = Rec("name", "igor", "year", 1981)
+  val test_record3 = { name: "quint", year: 2023, ...test_record }
   val test_set = Set(1, 2, 3)
```

Comment:
It looks like the grammar supports multiple splats, like `{...r1, ...r2, f:2}`. I see we error out in that case, but don't see a test for that. Could you add one?

Reply:
Yeah, I will add another test